src/main/java/org/opensearch/plugin/insights/core/service/QueryInsightsService.java

```diff
@@ -496,10 +508,14 @@ public QueryInsightsHealthStats getHealthStats() {
         Map<MetricType, TopQueriesHealthStats> topQueriesHealthStatsMap = topQueriesServices.entrySet()
             .stream()
             .collect(Collectors.toMap(Map.Entry::getKey, entry -> entry.getValue().getHealthStats()));
+        Map<String, Long> fieldTypeCacheStats = Optional.ofNullable(queryShapeGenerator)
+            .map(QueryShapeGenerator::getFieldTypeCacheStats)
+            .orElse(EMPTY_FIELD_TYPE_CACHE_STATS);
```

Comment:
Oh, I see how you using this `EMPTY_FIELD_TYPE_CACHE_STATS`. But in that case, why can't you use Java EMPTY map?

Reply:
I want to guarantee that `Map<String, Long> fieldTypeCacheStats` contains all 3 keys to avoid serialization issues & missing stats. It should return {hits:0, misses:0, bytes:0} instead of {}.

I think we can pass java empty map here and handle the null case one layer down in QueryInsightsHealthStats though. That will avoid the extra imports. I will change that.